### DISCUSSION TITLE
微信键盘 配色+布局

### DISCUSSION THREAD

**Discussion by laloe74** [imfuxiao/Hamster#560] [General]
微信键盘代码：
https://gist.github.com/laloe74/c0c2b481abb7f0f272a1b0c86cf67dc0

在线自定义工具（可一键导入当前配置）：
https://lost-melody.github.io/hamster-tools/

仓输入法官方教程：
https://github.com/imfuxiao/Hamster/wiki

完整[hamster.yaml](https://github.com/laloe74/rime-settings/blob/main/hamster.yaml)文件，可直接应用。

**Comment by GD2021** [comment]
不显示，中英切换按钮
